rotkehlchen/tests/unit/decoders/test_convex.py

```diff
@@ -999,3 +999,50 @@ def test_convex_claim_pending_rewards(database, ethereum_inquirer, ethereum_acco
         ),
     ]
     assert events == expected_events
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('ethereum_accounts', [['0xD18327BB6D6de9241Bed63bb5E78459325FbbD70']])
+def test_convex_withdraw_expired_lock(database, ethereum_inquirer, ethereum_accounts):
+    """
+    Tests a transaction that collects pending rewards but also compounds the pending CRV
+    in the pool. In this case the user is rewarded for performing this action.
+    """
+    user_address = ethereum_accounts[0]
+    evmhash = deserialize_evm_tx_hash('0x4b707540ec6eebf5e787c88df149e8141d2c295cda8a514da6d6111cf2deca40')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=ethereum_inquirer,
+        database=database,
+        tx_hash=evmhash,
+    )
+    timestamp = TimestampMS(1704839507000)
+    expected_events = [
+        EvmEvent(
+            tx_hash=evmhash,
+            sequence_index=0,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(amount=FVal('0.008929295372796715')),
+            location_label=user_address,
+            notes='Burned 0.008929295372796715 ETH for gas',
+            counterparty=CPT_GAS,
+            address=None,
+        ), EvmEvent(
+            tx_hash=evmhash,
+            sequence_index=363,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.WITHDRAWAL,
+            event_subtype=HistoryEventSubType.REMOVE_ASSET,
+            asset=A_CVX,
+            balance=Balance(amount=FVal('26633.698252368450151266')),
+            location_label=user_address,
+            notes='Unlock 26633.698252368450151266 CVX from convex',
+            counterparty=CPT_CONVEX,
+            address=string_to_evm_address('0x72a19342e8F1838460eBFCCEf09F6585e32db86E'),
```

Comment:
Use existing constant

```suggestion
            address=CVX_LOCKER_V2,
```